src/components/qa/subcomponents/QAContainer.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import ContainerQuestionEntryItem from './ContainerQuestionEntryItem';
+import { Typography } from '@material-ui/core';
```

Comment:
Module imports should always happen before component imports. Switch these imports around.